tests/Microsoft.Identity.Test.Common/TestConstants.cs

```diff
@@ -260,8 +260,22 @@ public static MsalTokenResponse CreateMsalTokenResponse()
                                                    ""id_token"": ""idtoken."",
                                                    ""client_info"": ""eyJ1aWQiOiI2ZWVkYTNhMS1jM2I5LTRlOTItYTk0ZC05NjVhNTBjMDZkZTciLCJ1dGlkIjoiNzJmOTg4YmYtODZmMS00MWFmLTkxYWItMmQ3Y2QwMTFkYjQ3In0""
                                                 }";
+
+        // Http Telemetry Constants
         public const string InteractiveRequestApiIdPlusCorrelationId = "1005,ad8c894a-557f-48c0-b045-c129590c344e,";
         public const string InteractiveRequestApiId = "1005";
+        public static Guid CorrelationId = new Guid("ad8c894a-557f-48c0-b045-c129590c344e");
+        public const string XClientCurrentTelemetryROPC = "2|1003,0|";
+        public const string XClientCurrentTelemetryROPCFailure = "2|1003,1003,1003,0|";
+        public const string XClientLastTelemetryROPC = "";
+        public const string XClientLastTelemetryROPCFailure =

```

Comment:
I would just keep these strings inline in the tests, I doubt these are reusable.